### ISSUE TITLE
Pager options

### ISSUE THREAD

**Issue by ramiromd** [creative-area/jquery.datagrid#1]
Hi all, i found this plugin today, and can say that is excelent, good job !. But, i have a problem trying to make the pagination widget. I am not using Bootstrap, i am using UiKit for the project, and i have defined the next plugin options:

```
$("#datagrid").datagrid({
    col:[ ... ],
    url: "radios/retrieve",
    paramsDefault:{
       page: 1,
       paging: 12
    },
    autoload: true,
    attr:{ "class": "uk-table uk-table-hover uk-table-striped uk-table-condensed"},
    pager:{
        item: "li",
        attrUl: {"class": "uk-pagination"},
        attrItemActive: { "class": "uk-active" },
        attrItemDisabled: { "class": "uk-disabled" },
        link: true,
        behavior: { "sliding": { "pages": 3 } },
        firstPage: "<<",
        prevPage: "<",
        nextPage: ">",
        lastPage: ">>"
   }
});
```

So, the table render is ok, but the pager is not renderized. The console show an error:

> [jquery.datagrid error] Unknown sorter: {"item":"li","attrUl":{"class":"uk-pagination"},"attrItemActive":{"class":"uk-active"},"attrItemDisabled":{"class":"uk-disabled"},"link":true,"behavior":{"sliding":{"pages":3}},"firstPage":"<<","prevPage":"<","nextPage":">","lastPage":">>"}

Why the error says "Unknown sorter" ? I am defining a pager, not a sorter. What are wrong with my code?.

Thanks !

**Comment by balda** [comment]
Hi!
Thanks for the issue: there was a typo in the error messages (corrected in 0.2.3).

What you get is a pager error. After updating in 0.2.3, you should see:
`[jquery.datagrid error] Unknown pager`.

For your pager config, you use an object. It must have only one key: the pager name (`default` in your case). So, your config should be:

``` javascript
$("#datagrid").datagrid({
    ...
    pager:{
        "default": {
            item: "li",
            attrUl: { "class": "uk-pagination" },
            attrItemActive: { "class": "uk-active" },
            attrItemDisabled: { "class": "uk-disabled" },
            link: true,
            behavior: { "sliding": { "pages": 3 } },
            firstPage: "<<",
            prevPage: "<",
            nextPage: ">",
            lastPage: ">>"
        }
    }
});
```

If you use UiKit frequently, make a plugin!

``` javascript
$.fn.datagrid( "plugin", "pager", "uikit", "default", {
    item: "li",
    attrUl: { "class": "uk-pagination" },
    attrItemActive: { "class": "uk-active" },
    attrItemDisabled: { "class": "uk-disabled" },
    link: true,
    behavior: { "sliding": { "pages": 3 } },
    firstPage: "<<",
    prevPage: "<",
    nextPage: ">",
    lastPage: ">>"
});
```

And just call it with:

``` javascript
$("#datagrid").datagrid({
    ...
    pager: "uikit"
});
```

**Comment by ramiromd** [comment]
Great ! Now works :)
